src/utils/db.js

```
@@ -0,0 +1,22 @@
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize(
+  process.env.PGDATABASE,
+  process.env.PGUSER,
+  process.env.PGPASSWORD,
```

Missing env validation: if any of the `process.env.*` variables (database, user, password, host) are undefined, Sequelize will be created with invalid values and connection errors may be hard to diagnose. Consider validating required env vars at startup and failing fast with a helpful error message if any are missing.